README.md

```diff
@@ -10,6 +10,13 @@ You can participate in this activity as long as you are college students over 18
 
 只要是你 18 周岁以上的大学生，就可以参加这个活动，你完全可以把它当作你的暑期实习，并且性价比和含金量都比一般实习还高。
 
+## What's gsoc-cn for?
+
+This organization is made up of previous GSoC student and would like to share their experience about how to apply and pass GSoC in here. If you find someone in here works for the same organization that you want to apply this year, feel free to contact them on [Telegram group](https://t.me/joinchat/B30CMA1YxJ0g0yo3GLj8uA) or IRC channel ([#gsoccn](https://www.irccloud.com/invite?channel=%23gsoccn&hostname=chat.freenode.net&port=6697&ssl=1) on freenode). I believe everyone in here is willing to tell you how to get a GSoC slot by writing a good proposal and negotiate with your community, but please be patient and polite when you ask for help :)
```

Comment:
..GSoC students who would like to share their experience of applying and passing GSoC. If you find members here interned for the organization(s) you want to apply for this year...